src/middlewared/middlewared/plugins/sysdataset.py

```diff
@@ -431,6 +435,8 @@ async def migrate(self, _from, _to):
 
         if await self.middleware.call('service.started', 'cifs'):
             restart.insert(0, 'cifs')
+        if await self.middleware.call('service.started', 'webdav'):
```

Comment:
Why do we want to restart `webdav` ?

Reply:
Apache keeps a log file open in `/var/log`, we need to stop it before umounting syslog dataset.